common/settings.h

```diff
@@ -126,6 +126,11 @@ struct Settings {
   // Whether the Dart VM service should be enabled.
   bool enable_observatory = false;
 
+  // Whether to publish the observatory URL over mDNS.
+  // On iOS 14 this prompts a local network permission dialog,
+  // which cannot be accepted or dismissed in a CI environment.
+  bool enable_observatory_publication = false;
+
```

Comment:
It seems like making the default value the value it was previously would be a safer change.

Reply:
Sure, I was copying the `enable_observatory` pattern right above this.